elastic/kibana
# [Custom threshold] Add log rate analysis to the alert details page for one document count aggregation

Closes #171163

## Summary

This PR adds log rate analysis to the alert details page for one document count aggregation if the user has a license above platinum. This is a similar implementation in the log threshold alert details page.

![image](https://github.com/elastic/kibana/assets/12370520/29cd29bf-ead5-4574-8121-739d3ed11fe7)

## 🧪 How to test?
- Create a Custom threshold rule with only one document count aggregation
    - Optional filter, document count aggregation filter, and group by information will be applied to the query of this component.
- Go to the alert details page, you should see the log rate analysis on this page
